functions/package.pp

```diff
@@ -0,0 +1,21 @@
+# @summary Generate package name that may or may not be in some SCL
+#
+# @param package
+#   The package name without any SCL prefix
+# @param scl
+#   An optional SCL name, without a dash as suffix
+#
+# @return The package name, with SCL prefix if applicable
+#
+# @example Without SCL
+#   katello_devel::package('postgresql-server') == ['postgresql-server']
+#
+# @example With SCL
+#   katello_devel::package('postgresql-server', 'rh-postgresql12') == ['rh-postgresql12-postgresql-server']
+function katello_devel::package(String[1] $package, Optional[String[1]] $scl = undef) >> String[1] {
```

Comment:
Would you mind copying (+ changing) the tests I wrote in the other PR as well?

Reply:
I did -- https://github.com/theforeman/puppet-katello_devel/pull/245/files#diff-88cf9c8f9f8eb64329bd90c44afc454a89dfa0acaa823380fcc8a5a3b6751164